examples/tech_meetings/20240403_using_enums_DIY.ipynb

```diff
@@ -0,0 +1,349 @@
+   "metadata": {},
+   "source": [
+    "In `ra2ce.analysis.analysis_config_data.enums`:\n",
+    "\n",
+    "- AnalysisDirectEnum,\n",
+    "- AnalysisIndirectEnum,\n",
+    "- DamageCurveEnum,\n",
+    "- EventTypeEnum,\n",
+    "- LossTypeEnum,\n",
+    "- RiskCalculationModeEnum,\n",
+    "- TripPurposesEnum,\n",
+    "- WeighingEnum"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ra2ce.analysis.analysis_config_data.enums.analysis_indirect_enum import AnalysisIndirectEnum\n",
+    "\n",
+    "for _indirect_type in AnalysisIndirectEnum:\n",
+    "    print(_indirect_type)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.2. Ra2ceEnumBase"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to normalize how we handle all our enumerations, a base class `Ra2ceEnumBase` (`ra2ce.configuration.ra2ce_enum_base`) is created (not that this __is not__ an [abstract class](https://docs.python.org/3/library/abc.html)).\n",
+    "\n",
+    "All our enumerations implement said base class and therefore __inherit__ its methods and definitions.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 2.2.1. Creating a new enum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can check its functionality by first creating our own enumeration here."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ra2ce.configuration.ra2ce_enum_base import Ra2ceEnumBase\n",
+    "\n",
+    "class DocumentationEnum(Ra2ceEnumBase):\n",
+    "    NONE = 0\n",
+    "    ALL = 1\n",
+    "    EXAMPLE = 2\n",
+    "    RESEARCH = 3\n",
+    "    TECH_MEETING = 4\n",
+    "    ARCHITECTURE = 5\n",
+    "    DOMAIN = 6\n",
+    "    INVALID = 99"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In our enumeration we see two sides, the name that we will use throughout the code (left side) and its associated value (right side). In short, we are just binding said names to a __unique__ value so that python handles everything internally whilst we can write 'readable' code.\n",
+    "\n",
+    "Some pointers for creation / modification of enums here:\n",
+    "\n",
+    "- Add new properties after the latest logical value. Sometimes you may find the last two values are `BANANA = 3` and `MIX = 100`, choose then to add the next value as `PINEAPPLE = 4` instead of `PINEAPPLE = 101`.\n",
+    "- Always define the names with capital letters.\n",
+    "- Try to use simplify the names whilst still being readable.\n",
+    "- Do not modify existing values unless previously discussed with the development team.\n",
+    "- `NONE=0` and `INVALID=99` are mandatory entries when inheriting from `Ra2ceEnumBase`."
```

Comment:
`NONE=0` is not mandatory and only added if the configuration attribute is optional.

Reply:
You are right, are read the definition in the base class wrongly. I corrected it.